tests/subsys/can/frame/src/main.c

```diff
@@ -77,6 +77,39 @@ static void test_zcan_frame_to_can_frame(void)
 		      "CAN msg length not same");
 }
 
+static void test_invalid_zcan_frame_to_can_frame(void)
+{
+	struct can_frame frame = { 0 };
+	struct can_frame expected = { 0 };
+	struct zcan_frame msg = { 0 };
+	const u8_t data[CAN_MAX_DLEN] = { 0x01, 0x02, 0x03, 0x04,
+					  0x05, 0x06, 0x07, 0x08 };
+
+	expected.can_id = BIT(30) | 0x678;
+	expected.can_dlc = sizeof(data);
+	memcpy(expected.data, data, sizeof(expected.data));
+
+	msg.rtr = 1U;
+	msg.id_type = 0U;
```

Comment:
Can you use `CAN_REMOTEREQUEST` and `CAN_STANDARD_IDENTIFIER` instead of numbers here?